components/kyma-environment-broker/internal/avs/internal_eval_assistant.go

```diff
@@ -0,0 +1,54 @@
+package avs
+
+import (
+	"strconv"
+
+	"github.com/kyma-incubator/compass/components/kyma-environment-broker/internal"
+	"github.com/kyma-incubator/compass/components/provisioner/pkg/gqlschema"
+)
+
+const (
+	evaluationIdKey = "avs_bridge.config.evaluations.cluster.id"
+	avsBridgeAPIKey = "avs_bridge.config.availabilityService.apiKey"
+)
+
+type InternalEvalAssistant struct {
+	avsConfig Config
+}
+
+func NewInternalEvalAssistant(avsConfig Config) *InternalEvalAssistant {
+	return &InternalEvalAssistant{avsConfig: avsConfig}
+}
+
+func (iec *InternalEvalAssistant) CreateBasicEvaluationRequest(operations internal.ProvisioningOperation, url string) (*BasicEvaluationCreateRequest, error) {
+	return newBasicEvaluationCreateRequest(operations, iec, iec.avsConfig.GroupId, url)
+}
+
+func (iec *InternalEvalAssistant) SetOverrides(inputCreator internal.ProvisionInputCreator, evaluationId int64) {
+	inputCreator.SetOverrides("avs-bridge", []*gqlschema.ConfigEntryInput{
```

Comment:
please use supported `AppendOverrides` function. `SetOverrides` has been deprecated, see [here](https://github.com/kyma-incubator/compass/blob/master/components/kyma-environment-broker/internal/model.go#L19).

Reply:
then I would think to rename also `SetOverrides` function from `EvalAssistant` interface to `AppendOverrides` to be consistent